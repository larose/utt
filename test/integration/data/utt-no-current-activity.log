2018-08-19 08:00 hello
2018-08-19 09:00 a

2018-08-20 08:00 hello
