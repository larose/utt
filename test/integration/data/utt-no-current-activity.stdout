
------------------------ Monday, Aug 20, 2018 (week 34) ------------------------

Working Time: 0h00
Break   Time: 0h00

----------------------------------- Projects -----------------------------------


---------------------------------- Activities ----------------------------------



----------------------------------- Details ------------------------------------


